packages/shared/src/modules/reviews/libs/types/review-get-all-response-dto.type.ts

```diff
@@ -0,0 +1,15 @@
+type ReviewGetAllResponseDto = {
+	content: string;
+	id: number;
+	likesCount: number;
+	poiId: null | number;
+	routeId: null | number;
+	user: {
+		avatar: null | { url?: null | string };
```

Comment:
let's return either string or null
```suggestion
		avatarUrl: null | string;
```

Reply:
It’s a derived property that we build in the service layer by mapping avatar.url into a flat avatarUrl field for the dto